holoviews/pyodide.py

```diff
@@ -17,6 +17,12 @@
 #-----------------------------------------------------------------------------
 _background_task = set()
 
+try:
+    import pyodide_http
+    pyodide_http.patch_all()
+except Exception:
+    pass
```

Comment:
Does not seem needed, import `panel.io.pyodide` will already do that.

Reply:
You are right.

Reply:
I got a bit confused as to why I couldn't download files in Jupyterlite, but it was a cors issue.